### PR TITLE
You can't see it, but I'm making a very angry face right now

### PR DESCRIPTION
**Pull request by Yoghurt4C**
changes:
-updated loom to shorten compile time (and enlarge logspam)
-move renderer registration out of the modelappender (fixes https://github.com/williewillus/botania-fabric-issues/issues/28
-added hard dependencies to fabric.mod.json
-mostly prevented the lightningsegmentgenerator from crashing
-made lightning fx render again

### CODE REVIEW

File: src/main/resources/fabric.mod.json [hunk 1]
```diff
@@ -39,7 +39,10 @@
   "depends": {
     "fabricloader": ">=0.8",
     "fabric": ">=0.13",
-    "minecraft": ">=1.16.2"
+    "minecraft": ">=1.16.2",
+    "libblockattributes": ">=0.8.4",
+    "curios" : "*",
+    "patchouli": "*"
```
Comment (williewillus) [inline src/main/resources/fabric.mod.json:45]: minecraft  to 1.16.3 and fapi to the one I use in build.gradle

**Review by williewillus — CHANGES_REQUESTED** [review]
angerypotato

**Comment by Yoghurt4C** [comment]
image

**Comment by Alwinfy** [comment]
dibs on the water-type

**Comment by williewillus** [comment]
Can you make the same lightning changes on forge?

**Pull request merged**